orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/quip/VerifyQuipTask.groovy

```diff
@@ -45,19 +45,18 @@ class VerifyQuipTask extends AbstractQuipTask implements Task {
     String cluster = stage.context?.clusterName
     String region = stage.context?.region
     String account = stage.context?.account
-    String app = stage.context?.application
     Map instances = stage.context?.instances
     ArrayList healthProviders = stage.context?.healthProviders
     Map stageOutputs = [:]
     ExecutionStatus executionStatus = ExecutionStatus.SUCCEEDED
-    if (cluster && region && account && healthProviders != null && app && instances) {
+    if (cluster && region && account && healthProviders != null && instances) {
```

Comment:
elsewhere we are deriving app from cluster name, so I've taken it out where it was being enforced as a required context parameter